tests/admin_views/admin.py

```diff
@@ -450,8 +452,11 @@ def multiline(self, instance):
 
     def multiline_html(self, instance):
         return mark_safe("Multiline<br>\nhtml<br>\ncontent")
-    multiline_html.allow_tags = True
 
+    def multiline_html_allow_tags(self, instance):
+        return "Multiline<br>html<br>content<br>with allow tags"
+
+    multiline_html_allow_tags.allow_tags = True
     value.short_description = 'Value in $US'
```

Comment:
while you are here, could you move this below the `value` method - it seems to be orphaned from its parent.